discovery/discovery.go

```diff
@@ -148,12 +148,12 @@ func doDiscover(app App, pre string, insecure bool) (*Endpoints, error) {
 // DiscoverEndpoints will make HTTPS requests to find the ac-discovery meta
```

Comment:
@jonboulle You're right. I tried to summarize the various doubts (and the already opened issues about discovery and label matching) in #140.
Feel free to close this and the related PR (also on coreos/rocket) as I think that this is going to be outdated.